src/Twig/AppExtension.php

```diff
@@ -130,6 +130,7 @@ public function getFunctions()
             new TwigFunction('call_user_func', array($this, 'callUserFunc')),
             new TwigFunction('emsco_generate_email', array($this, 'generateEmailMessage')),
             new TwigFunction('emsco_send_email', array($this, 'sendEmail')),
+            new TwigFunction('emsco_users_by_role_and_circles', [UserService::class, 'getUsersForRoleAndCircles']),
```

Comment:
This is exposing a DB query directly from the repository directly in TWIG.
This is also very specific.

You could create a dedicated runtime UserRuntime that creates a UserList object. This object should have the functionality to filter based on role and/or circles. This is much more flexible for future userList features. 

Possible feature request: only show the enabled users (from the full UserList)

Reply:
👍